typescript/framework-extensions/vercel-ai-sdk/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@coinbase/cdp-vercel-ai-sdk",
```

```suggestion
  "name": "@coinbase/agentkit-vercel-ai-sdk",
```

Please update all docs and code samples to use the updated package name